src/Standards/Generic/Tests/Arrays/DisallowCompactArrayBuilderUnitTest.2.inc

```diff
@@ -0,0 +1,22 @@
+<?php
+$var = compact('s', 'b');
+
+// The following function has a simulated git conflict for testing.
+// This is not a merge conflict - it is a valid test case to test handling of
+// compact function call without associated closer.
+// Please do not remove.
+function test()
```

Comment:
I'm not sure why this test is included ?

Reply:
Fair enough, will remove. (I've copied some testes from different sniff as I started to work on it).